terraform/eks/e2e/main.tf

```diff
@@ -134,4 +149,17 @@ resource "null_resource" "validator" {
       -sample_app="${var.test_dir}/${var.sample_app}"
     EOT
   }
+
+  provisioner "local-exec" {
+    when    = destroy
+    command = <<-EOT
+      echo "Running cleanup for K8s resources"
+      go test -timeout 30m -v ${self.triggers.test_dir} \
+      -destroy=true \
+      -region=${self.triggers.region} \
+      -eksClusterName=${self.triggers.cluster_name} \
+      -computeType=EKS \
+      -eksDeploymentStrategy=DAEMON
```

Comment:
What's the alternative strategy here ?

Reply:
`REPLICA` or `SIDECAR` are the current options. I can add a `STATEFUL` type too and make this configurable for future testing. I didn't include it now because we don't change the deployment strategy from the default as of now.